lib/commands/start.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Commands
+  class Start < Base
+    def call
+      bot.api.send_message(chat_id: chat_id, text: Locale.t('commands.start_message'))
```

Comment:
if we go on with the idea of having templating system, `t` could be added as a helper to the renderer and here we're gonna use a very simple template:

```erb
<%= t('commands.start_message') %>
```

Thus we unify the way of working with messages (via templates).